client/src/components/Forms.js

```diff
@@ -206,9 +215,37 @@ const GoalForm = (props) => {
             <Input  onChange={props.onChange} type="date" name="raceDate" id="raceDate"/>
           </FormGroup>
         </Form>
-      </CardBody> 
       </div>
     </div>
   );
 }
- 
\ No newline at end of file
+const ContactForm = (props) => {
+  return ( 
+    <div class="card">
+          <div class="card-header">Contact</div>
+          <div class="card-body">
+            <Form>
+              <div class="form-group">
+                <label for="exampleFormControlInput1">Your Name</label>
+                <Input type="name" class="form-control" id="exampleFormControlInput1" placeholder=""></Input>
```

Comment:
If these need to keep their IDs let's name them something descriptive. A few of these elements have the same ID.

Reply:
Done